code/modules/vehicles/mecha/equipment/tools/medical_tools.dm

```diff
@@ -236,6 +236,12 @@
 	var/mob/living/carbon/M = patient
 	if(!M)
 		return
+	if(M.loc != src)
+		to_chat(chassis.occupants, "[icon2html(src, chassis.occupants)][span_notice("[patient] no longer detected. Life support functions diabled.")]")
+		log_message("[patient] no longer detected - Life support functions disabled.", LOG_MECHA)
+		STOP_PROCESSING(SSobj, src)
+		patient = null
+		update_equip_info()
```

Comment:
Rather then checking to see if the patient isn't in on process, couldn't you do it in Exited?

Reply:
If they get teleported out of it for whatever reason, such as bluespace dust, it doesn't call `go_out`, which is where the bugs in the report and in the description come into play.